rpcs3/Emu/RSX/Program/CgBinaryVertexProgram.cpp

```diff
@@ -174,6 +174,7 @@ std::string CgBinaryDisasm::GetSRCDisasm(const u32 n)
 	case 0: abs = d0.src0_abs; break;
 	case 1: abs = d0.src1_abs; break;
 	case 2: abs = d0.src2_abs; break;
+	default: fmt::throw_exception("n out of bounds (n=%d)", n);
```

Comment:
assume(0) and move the assert to the start of the func

Reply:
done